server/src/main/resources/org/elasticsearch/tasks/task-index-mapping.json

```diff
@@ -19,7 +19,7 @@
           "id": {
             "type": "long"
           },
-          "parent_id": {
+          "parent_task_id": {
```

Comment:
I think we need to increment the version in `TaskResultsService.TASK_RESULT_MAPPING_VERSION` in order to get the new mapping applied.

Reply:
good point, I was not aware of that. Do you know if there are tests around applying the updated mappings?

Reply:
I think not, since there would at least be no verification that the parent_task_id field is in the index after/during a rolling upgrade. I also seem to remember that some of the test frameworks delete the .tasks index between each test (but not sure if it applies to all upgrade tests).